lib/public_key/src/pubkey_crl.erl

```diff
@@ -681,7 +681,7 @@ status(Reason) ->
     {revoked, Reason}.
 
 verify_extensions([#'TBSCertList_revokedCertificates_SEQOF'{crlEntryExtensions = Ext} | Rest]) ->
-    verify_extensions(pubkey_cert:extensions_list(Ext)) and verify_extensions(Rest);
+    verify_extensions(pubkey_cert:extensions_list(Ext)) andalso verify_extensions(Rest);
```

Comment:
This is also a change in behavior that may lead to `verify_extensions` behaving differently. Assume that `Rest` contains some garbage, like `foo`. With `and`, `verify_extensions` would _always_ throw an exception in that case. With `andalso`, it will _not_ throw an exception (but return `false`) if any value before lets it result in `false`.

This tl;dr means that the right-hand sides of updated boolean operators need to be checked for side effects (which could be an exception). While they would _always_ happen with `and` or `or`, they may not happen any more with `andalso` or `orelse` because of their short-circuit behavior.